File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.kt
class Solution {
    fun runningSum(nums: IntArray): IntArray {
        val res = IntArray(nums.size) { nums[0] }
        for (i in 1..(nums.size-1)) {
            res[i] = nums[i] + res[i-1]
        }
        return res
    }
}

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        
        res = []
        
        for i, n in enumerate(nums):
            if i == 0:
                res += [n]
            else:
                res += [n + res[-1]]
                
        return res